src/app/layout.js

```diff
@@ -3,9 +3,10 @@ import { Route, Switch, Redirect } from 'react-router-dom';
 import { ConnectedRouter } from 'connected-react-router';
 
 import { history } from 'redux/store';
-import { CALCULATOR } from 'constants/routes';
+import { CALCULATOR, RECORD } from 'constants/routes';
 
 import Topbar from './components/Topbar';
+import RecordLayout from './screens/Record';
 import CalculatorLayout from './screens/Calculator';
```

Comment:
Instead of `RecordLayout` and `CalculatorLayout`, call the `Record` and `Layout`